src/northbridge/amd/pi/00730F01/northbridge.c

```diff
@@ -36,6 +36,7 @@
 #include <arch/acpigen.h>
 #include <northbridge/amd/pi/nb_common.h>
 #include <northbridge/amd/agesa/agesa_helper.h>
+#include "bios_knobs.h"
```

Comment:
@mwit-3mdeb please include full path to bios knobs. We do not want the compiler to search for some random header file
```suggestion
#include "mainboard/pcengines/apu2/bios_knobs.h"
```